packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -350,6 +380,7 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       ...this.state,
       xScale,
       yScale,
+      yScaleSecondary,
```

Comment:
i dont think this line is needed

Reply:
This line is needed, as it passes the secondary y scale so that it can be used for creating the lines. If this line is removed, the lines can no longer use the secondary y scale.